### ISSUE TITLE
[Multi-Actions] Figure out generic action configurations

### ISSUE THREAD

**Issue by spolu** [dust-tt/dust#4597]
Reference: https://www.notion.so/dust-tt/Multi-Actions-Technical-Proposal-662aa8afbc584f4285d89becc0a6417b

This one requires a bit of framing

The shared action configurations fields are:
- `description`
- `force_use_at_iteration`

Maybe `max_uses_per_run`, `min_uses_per_run`

We need to figure out where we will store that. I presume the simplest is that it will exist on all action configuration models.

As part of this, add these new fields to all existing action configuration (including generation)

Figure out what are the invariants in naming that cover existing actions, generation and future one. What are the base classes in code? BaseActionResource likely?

**Comment by fontanierh** [comment]
TLDR:
- just adding `force_use_at_iteration` for now

**Comment by fontanierh** [comment]
It's the only one we really for backward compatibility. We'll def want `description` later on. Possibly `max_uses_per_run` and `min_uses_per_run` 👍

**Comment by spolu** [comment]
👍
